spec/Vehicle/Vehicle.vspec

```diff
@@ -399,10 +399,48 @@ CurrentLocation.Altitude:
   datatype: double
   type: sensor
   unit: m
-  description: Current altitude relative to WGS 84 reference ellipsoid.
+  description: Current altitude relative to WGS 84 reference ellipsoid, as measured at position of GNSS receiver antenna.
 
 CurrentLocation.VerticalAccuracy:
   datatype: double
   type: sensor
   unit: m
   description: Accuracy of altitude.
+
+CurrentLocation.GNSSReceiver:
+  type: branch
+  description: Information on the GNSS receiver used for determining current location.
+
+CurrentLocation.GNSSReceiver.Status:
```

Comment:
In SENSORIS, the status is compromised of multiple attributes, with the fix_type being just one of them. Others are e.g. number of satellites used for fix, DOP values, SNR. Therefore I would recommend naming the attribute FixType.